src/scss/_objects.accordion.scss

```diff
@@ -0,0 +1,17 @@
+.accordion {
+  background-color: #f6f6f6;
```

Comment:
@phuson I've spoke with @kylechristopherb about this. They want the background color to always be gray (`#F6F6F6`).

Should I handle this with some sort of theme prop?